package.json

```diff
@@ -28,7 +28,7 @@
     "prepush": "yarn test && yarn test:routes"
   },
   "dependencies": {
-    "@hmcts/cmc-common-frontend": "^1.40.0",
+    "@hmcts/cmc-common-frontend": "file:/Users/robertparkinson/git/MOJ/cmc-common-frontend",
```

Comment:
i thought that if you did 
`yarn link @hmcts/cmc-common-frontend`
It didn't touch the package.json? It just removes the current one and adds a symlink

Reply:
I'll have to give that a try. this has been the way I've always done it :)